setup.py

```diff
@@ -88,6 +88,10 @@
         "datasets<=2.16",
         "transformers>=4.26,<4.38",
     ],
+    "exporters-executorch": [
+        "executorch>=0.4.0",
+        "transformers>=4.46",
+    ],
```

Comment:
@michaelbenayoun I looked into one CI failure related to ExecuTorch https://github.com/huggingface/optimum/actions/runs/12171925051/job/33984832982?pr=2090#step:5:50
```
ValueError: The minimum Transformers version compatible with ExecuTorch is 4.46.0. Please upgrade to Transformers 4.46.0 or later.
```
The failure is because the required version is not installed in the runner. Here is where I specify the dep packages and versions. Do I need to modify somewhere else to force the install the required package?